js/index.js

```diff
@@ -1,2 +1,2 @@
-import { module } from '../crate/src/lib.rs'
```

Comment:
Maybe we could use a similar technic that we do on webpack; watching the .rs files oursevles.

@catsigma would you be interested in joing us on a video chat?

Reply:
I am in

Reply:
The plugin has been updated to `1.2.4` which enables:
```
import module from '../crate/src/lib.rs'
module.run()
```
or
```
import { run } from '../crate/src/lib.rs'
run()
```
or 
```
import module from '../crate/Cargo.toml'
module.run()
```
or
```
import { run } from '../crate/Cargo.toml'
run()
```